rimage/image_file.go

```diff
@@ -259,7 +259,14 @@ func DecodeImage(ctx context.Context, imgBytes []byte, mimeType string) (image.I
 	case "", ut.MimeTypeJPEG:
 		img, err := DecodeJPEG(bytes.NewReader(imgBytes))
 		if err != nil {
-			return nil, err
+			if mimeType == "" {
+				img, _, err = image.Decode(bytes.NewReader(imgBytes))
+				if err != nil {
+					return nil, err
+				}
+			} else {
+				return nil, err
+			}
```

Comment:
I know we wrote this together, but actually I think what you can do is just put this in its own `case` statement, so that you don't have to check again if the mimeType is ""

so it would be 
```
case "":
    img, err := DecodeJPEG(bytes.NewReader(imgBytes))
    if err != nil {
        img, _, err = image.Decode(bytes.NewReader(imgBytes))
        if err != nil {
            return nil, err
       }
    }
    return img, nil   

```